app/jobs/socure_docv_results_job.rb

```diff
@@ -3,22 +3,30 @@
 class SocureDocvResultsJob < ApplicationJob
   queue_as :default
 
-  attr_reader :document_capture_session_uuid
+  attr_reader :document_capture_session_uuid, :async
 
   # @param [String] document_capture_session_uuid
-  def perform(document_capture_session_uuid:)
+  def perform(document_capture_session_uuid:, async: true)
     @document_capture_session_uuid = document_capture_session_uuid
+    @async = async
 
-    dcs = DocumentCaptureSession.find_by(uuid: document_capture_session_uuid)
-    raise "DocumentCaptureSession not found: #{document_capture_session_uuid}" if !dcs
+    raise "DocumentCaptureSession not found: #{document_capture_session_uuid}" unless
+      document_capture_session
 
-    # analytics = create_analytics(
-    #   user: dcs.user,
-    #   service_provider_issuer: dcs.issuer,
-    # )
+    @analytics = create_analytics(
+      user: document_capture_session.user,
+      service_provider_issuer: document_capture_session.issuer,
+    )
```

Comment:
```suggestion
```

how about replacing the  `create_analytics` method with:

def analytics
  @analytics ||= Analytics.new(
      user: document_capture_session.user,
      service_provider_issuer: document_capture_session.issuer,
    )
end

👍🏿